### ISSUE TITLE
Space after echoed vertical line is trimmed

### ISSUE THREAD

**Issue by SiarheiKuchukBelitsoft** [alexkrechik/VSCucumberAutoComplete#321]
**Describe the bug**
Space after echoed vertical line is trimmed

**To Reproduce**
1. Create file and put following lines
```
        Then  List of Enrollment Requests should be following
              | column 1 | column 2 |
              | C#1         | Mama \| Mia |
```
2. Press save

Actual:
```
              | column 1 | column 2 |
              | C#1         | Mama \|Mia |

```
Expected:
```
              | column 1 | column 2 |
              | C#1         | Mama \| Mia |
```

**Comment by SiarheiKuchukBelitsoft** [comment]
Hello

I've updated. Please pay attention to escaping vertical line
`\|`

**Comment by alexkrechik** [comment]
Result in the 3.0.0 version:
<img width="471" alt="Screenshot 2024-05-15 at 21 13 37" src="https://github.com/alexkrechik/VSCucumberAutoComplete/assets/5772406/6a86f228-413d-433b-9b42-9241d1430d90">

**Comment by drweb86** [comment]
BTW what are your thoughts about adding support of cucumber+markdown files. Did it change?

My vision has evolved over time of using of cucumber for integration-ui tests.
1. integration-ui tests are not a viable option anymore. if someone needs to write a integration-ui test, he should write a unit-test instead. Unit-tests are faster to run and they are more stable. In company where i work for we don't automate this staff anymore.
2. feature files are to be knowledge storage, but should knowledge only be stored in algorithm like staff - its a good question. Business people never read BDD, it was kind of fake advertisement of this thing, QA reads scenarios and they are used to them.

**Comment by alexkrechik** [comment]
This issue is to be fixed or implemented in the 3.0.0 version of the extension. If this issue is unresolved or there are additional notes, please create a new ticket with the necessary details.
